tests/kyma/fixtures/kubeconfig.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: v1
```

Comment:
probably this shouldn't be added to the PR

Reply:
Solved